parts/linux/cloud-init/artifacts/components.json

```diff
@@ -116,7 +116,7 @@
         },
         {
           "renovateTag": "registry=https://mcr.microsoft.com, name=containernetworking/azure-cns",
-          "latestVersion": "v1.6.10",
+          "latestVersion": "v1.6.13",
           "previousLatestVersion": "v1.6.7",
```

Comment:
we probably want to keep 1.6.10 as the previous latest

Reply:
I believe 1.6.10 release was paused and almost all of prod remains on 1.6.7

Reply:
correct his never went out 1.6..7 should be previous.